bigbluebutton-html5/imports/ui/components/whiteboard/component.jsx

```diff
@@ -103,71 +103,72 @@ export default function Whiteboard(props) {
     let stack = null;
     let changed = false;
 
-    if (next.pageStates[curPageId] && !_.isEqual(prevShapes, shapes)) {
-      // mergeDocument loses bindings and history, save it
-      pageBindings = tldrawAPI?.getPage(curPageId)?.bindings;
-      history = tldrawAPI?.history
-      stack = tldrawAPI?.stack
-
-      next.pages[curPageId].shapes = shapes;
+    if (curPageId) {

```

Comment:
Would you consider
` if (!currPageId) return currentDoc;`
instead of the 60 lines of indentation?